src/main/kotlin/DetektDemo.kt

```diff
@@ -1,2 +1,55 @@
 class DetektDemo {
-}
\ No newline at end of file
+    private val unusedVariable: String = "unused"
+
+    fun isHTTPSuccess(statusCode: Int): Boolean = statusCode == 200
+
+    fun veryLongFunction() {
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello")
+    }
+
+    fun fixMyStyle(x: Int): Int {
+        when (x) {
+                1 -> {
+                println(x)
+                return x
+            }
+            2 -> {
+                println(x)
+                return x
+            }
+            else -> { throw Exception() }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/50ce9000fbc0665351fc0cbb11f07ed8a3b0b789/src/main/kotlin/DetektDemo.kt#L34"></span>Exception is a too generic Exception. Prefer throwing specific exceptions that indicate a specific error case.</td>
    </tr>
  </tbody>
</table>